### DISCUSSION TITLE
Welcome to Modchat Discussions!

### DISCUSSION THREAD

**Discussion by micahlt** [micahlt/modchat#43] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about
  * Share feature ideas and requests
  * Engage with other community members

  To get started, comment below with an introduction of yourself and tell us about how you use Modchat!

**Comment by jessiejs** [comment]
Well, that was embarrising

My qublip Github was flagged, meaning that you can't see any of my pull requests. Shoot.
_pew pew, pewty pew pew pew._ 
_Ah! I have been shot. lol_

**Comment by jessiejs** [comment]
Lol, I just took down modchat..... by posting possibly the most dangerous post ever, carefully designed to take down any server. "Hello". Lol

**Reply by jessiejs** [reply]
A-also, could you make it so you can post from the previous two sign in's that you used, because I have multiple computers, and I have to sign back into modchat. Every. Single. Time.

**Comment by shrysjain** [comment]
Hi....I opened two pull req's. Its kind of confusing but let me explain here as well:
Create a new folder in public called 'appeals'
Move appeals.html to that folder and rename it index.html
Move appeals.css to the folder

This is if ModChat needs unban appeal information cuz im hearing about some ban drama happening

**Comment by jessiejs** [comment]
Lol, deadness go brrrrrrrrrrrrrrrr
